packages/common/src/local-data/local-data.ts

```diff
@@ -7,10 +7,10 @@ let _localDataDir = "";
 export const getMeticulousLocalDataDir: () => string = () => {
   const logger = log.getLogger(METICULOUS_LOGGER_NAME);
   if (!_localDataDir) {
-    logger.warn(
-      "Local data dir has not been set by setMeticulousLocalDataDir()!"
-    );
     setMeticulousLocalDataDir();
+    logger.debug(
```

Comment:
I wonder if we'd be fine with just the log in line 15? I'm fine either way.

Reply:
I'll make it an else